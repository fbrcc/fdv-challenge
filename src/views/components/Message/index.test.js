import React from 'react';
import { Message } from './index'
import renderer from 'react-test-renderer';

it('renders correctly', () => {
    const tree = renderer
      .create(<Message visitors={[{name: "Fermin", country: "Argentina", date: "1989-09-18"}]} selected={0}></Message>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });