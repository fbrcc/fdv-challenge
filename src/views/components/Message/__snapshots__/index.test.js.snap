// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <p
    className="sc-bdVaJa fOgzJu"
  >
    Hola 
    Fermin
     de 
    Argentina
    . Hoy es día 
    02
     del mes 
    03
     y tenes 
    28
     años.
  </p>
</div>
`;
